ignite/pkg/cliui/view/accountview/account_test.go

```diff
@@ -0,0 +1,54 @@
+package accountview
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccount_String(t *testing.T) {
```

Comment:
Can you please follow the current pattern we have for the table tests??

https://github.com/ignite/cli/blob/728e82d883b9f26d63a5ebe7c0d3bdea7e6eb60c/ignite/pkg/cosmosanalysis/app/app_test.go#L365-L365

The subtests (`t.Run`) should be called for each test case, and we can pass the name of the test in the struct.

```go
	tests := []struct {
		name      string
		account  Account
		want       string
		empty     bool
	}{
		// cases...
	}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			// tests...
		})
	}
```

This way, we can also avoid having two table tests in the same function.

Reply:
done. changed it to include the test name in the struct as you suggested.